language_detector/main.py

```diff
@@ -2,10 +2,39 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    words = text.split()
+    count_dict ={}
+    
+    def word_count(words,common_words):
+        """ Returns the common word count given the list of words in a given text. """
+        count = 0
+        for word in words:
+            if word in common_words:
+                count+=1
+        return count
+        
+    for elem in LANGUAGES:
+        """ Loops through each language and calculates the corresponding common word count. """
```

Comment:
Thanks for the comments and notes. Our group approached this problem differently and I was able to learn how your team solved the problem with ease due to your comments. Good job